contributors.html

```diff
@@ -137,7 +137,7 @@ <h3>Links</h3>
                 <h3> ` + element.login + ` </h3>
                 <p> ` + element.contributions + ` commits </p>
                 <ul class="probootstrap-footer-social">
-                  <li class="facebook"><a href="` + element.html_url + `"><i
+                  <li class="facebook"><a href="` + element.html_url + `" target="_blank"><i
```

Comment:
> rel="noopener noreferrer" should be added to links containing target="_blank" as a precaution against reverse tabnabbing


So also add `rel="noopener noreferrer"` to this line.

Reply:
ok